src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
```

Comment:
I just tested it and it fails for smaller text and passes for larger text. At this point, I'd like to point out I didn't pick this color (although of course I love it) and white text provides the greatest possible amount of contrast. :) Orange says "warning" to me, so it seems like a natural fit for this pattern.